testing/compareResults.py

```diff
@@ -0,0 +1,73 @@
+import sys
+import os
+import matplotlib.pyplot as plt
+import numpy as np
+
+def openFoam_results_reader(path_to_file):
+    velocity_values = []
+    with open(path_to_file) as f:
+        f = f.readlines()
+        in_value_list = False
+        line = 0
+        # detect beginning of mesh velocity values
+        while not in_value_list:
+            line_content = f[line]
+            if line_content == '(\n':
+                in_value_list = True
+            line += 1
+        i = 0
+        # extract velocities from openFoam results
+        while in_value_list:
+            line_content = f[line]
+            if line_content == ')\n': # exit at end of values list
+                break
+                in_value_list = False
+            line_values = line_content[1:len(line_content) - 2].split(' ')
+            velocity_values.append([float(line_values[0]), float(line_values[1]), float(line_values[2])])
+            line += 1
+            i += 1
+    return(velocity_values)
+
+def cppfd_results_reader(path_to_file):
+    velocity_values = []
+    with open(path_to_file) as f:
+        f = f.readlines()
+        n_lines = len(f)
+        # extract velocities from
+        for i in range(n_lines):
+            line_content = f[i]
+            line_values = line_content[:len(line_content) - 1].split(' ')
+            velocity_values.append([float(line_values[0]), float(line_values[1])])
+    return(velocity_values)
+
+def main():
+    # input paths to result files and size of square mesh
+    path_to_openFoam_file = str(sys.argv[1])
+    path_to_cppfd_file = str(sys.argv[2])
+
+    # extract velocity values from openFoam results
+    openFoam_velocity_values = openFoam_results_reader(path_to_openFoam_file)
+    N_oF = int(round(np.sqrt(len(openFoam_velocity_values))))
```

Comment:
STYLE: be consistent, we do not capitalize variable letters except for proper nouns. That should be `n_OF` or `n_of` but not `N_oF`